physx.js

```diff
@@ -807,6 +807,51 @@ const physxWorker = (() => {
       z
     );
   };
+
+  w.getPathPhysics = (physics, start, dest, hy, heightTolerance, detectStep, maxIterdetect, maxIterStep, maxVoxelCacheLen, ignorePhysicsIds) => {

```

Comment:
IMO this should just take 2 arguments, the start and end.

A-star does not have any other parameters that are not already present in the scene.

It is especially troubling to see a "cache length" here; I still have no idea what that is, or what I should be inputting it since it has nothing to do with A-star.

I also don't think ignore physics ids should be here, since already have 3 other methods for setting PhysX flags, and the algorithm can check that.